tests/unit/test_helpers.py

```diff
@@ -208,7 +208,7 @@ def _compare_entity_proto(entity_pb1, entity_pb2):
             assert val1 == val2
 
 
-def test_enity_to_protobf_w_empty():
+def test_entity_to_protobf_w_empty():
```

Comment:
Typo for next time: protobf -> protobuf .

Reply:
Addressed in https://github.com/googleapis/python-datastore/pull/525/commits/dc54502f83989a20f45bc9673ba7ae0dcbdbce56